dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -24,7 +27,8 @@
     populate_job_error_info, get_action_dates
 )
 
-from dataactcore.models.domainModels import matching_cars_subquery, Office, concat_tas_dict, concat_display_tas_dict
+from dataactcore.models.domainModels import concat_tas_dict_vectorized
+from dataactcore.models.domainModels import Office, concat_display_tas_dict
```

Comment:
merge?

Reply:
ah, good point. Probably a result of git merges. Not sure why flake didn't flake-out on that. Fixing.